### ISSUE TITLE
GEOS : Weekly Team Meeting Agenda 2022

### ISSUE THREAD

**Issue by PLoiya** [hackforla/GreenEarthOS#100]
### Overview
We need an agenda for the coming meeting and notes about what happens during the meetings so we can stay on track

### Action Items
- [ ] Add items for the agenda to comments at the bottom, or into the next week's agenda if it's available
- [ ] Add link to agenda's past to the Resources section below
- [ ] Create new agendas for the team by copying template below

```
## Date Agenda and notes

### Prework
- [ ] 

### Roll Call
- [ ] Bonnie
- [ ] Puneet
- [ ] Joanna

#### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/GreenEarthOS/projects/1#column-15327305)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Sweta](https://github.com/hackforla/GreenEarthOS/issues/assigned/ssweta)
       - [ ] [Joanna]()


#### New Items
- [ ] 
- [ ] Carry forward anything we didn't have time for 

### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

```

### Prior meetings
[2022-01-19](https://github.com/hackforla/GreenEarthOS/issues/100#issuecomment-1017077194)

**Comment by PLoiya** [comment]
Updated - 19-Jan-2022

Agenda for 12/21/21

### Prework
- [ ] issue for audit #31

### Roll Call
Bonnie, Puneet

#### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/GreenEarthOS/projects/1#column-15327305)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [x] [Sweta](https://github.com/hackforla/GreenEarthOS/issues/assigned/ssweta)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/GreenEarthOS/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further

#### New Items
- [x] change label from climate events to climate conferences
- [ ] Did Sweta make her issue, and do any updates on the wiki?  If not, carry over to next meeting
- [ ] Did Bonnie update the audit issue, and make a second issue specifically for Climate Conferences issues

#### Notes
We spent the meeting 1.75 hours sorting out the issues for the climate conference bucket.
Everything else will be carried over until the next meeting

#### Task Items

**Comment by ExperimentsInHonesty** [comment]
## Date Agenda and notes

### Prework
- [ ] 

### Roll Call
- [ ] Bonnie
- [ ] Puneet
- [ ] Joanna

#### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/GreenEarthOS/projects/1#column-15327305)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Sweta](https://github.com/hackforla/GreenEarthOS/issues/assigned/ssweta)
       - [ ] [Joanna](https://github.com/hackforla/GreenEarthOS/issues/assigned/joval604)
       - [ ] [Michelle](https://github.com/hackforla/GreenEarthOS/issues/assigned/miolow)
       - [ ] [Vaisali](https://github.com/hackforla/GreenEarthOS/issues/assigned/vaisali89)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/GreenEarthOS/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further

#### New Items
- [ ] 
- [ ] Carry forward anything we didn't have time for 

### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

**Comment by joval604** [comment]
### Overview
We need an agenda for the coming meeting and notes about what happens during the meetings so we can stay on track

### Action Items
- [ ] Add items for the agenda to comments at the bottom, or into the next week's agenda if it's available
- [ ] Add link to agenda's past to the Resources section below
- [ ] Create new agendas for the team by copying template below

```
## Date Agenda and notes

### Prework
- [ ] 

### Roll Call
- [ ] Bonnie
- [ ] Puneet
- [ ] Joanna

#### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/GreenEarthOS/projects/1#column-15327305)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Sweta](https://github.com/hackforla/GreenEarthOS/issues/assigned/ssweta)
       - [ ] [Joanna]()
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/GreenEarthOS/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further

#### New Items
- [ ] 
- [ ] Carry forward anything we didn't have time for 

### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

```

### Resources/Instructions

### Prior meetings
[2022-01-19](https://github.com/hackforla/GreenEarthOS/issues/100#issuecomment-1022805847)

**Comment by joval604** [comment]
## 2022-02-16 Agenda and notes

### Roll Call
- [] Bonnie
- [ ] Puneet
- [] Joanna
- [ ] Vaisali
- [ ] Michelle
- [] Sanya (Observing)

#### Recurring items: 
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each team member
       - [] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Sweta](https://github.com/hackforla/GreenEarthOS/issues/assigned/ssweta)
       - [] [Joanna](https://github.com/hackforla/GreenEarthOS/issues/assigned/joval604)
       - [ ] [Michelle](https://github.com/hackforla/GreenEarthOS/issues/assigned/miolow)
       - [ ] [Vaisali](https://github.com/hackforla/GreenEarthOS/issues/assigned/vaisali89)

#### New Items
- [ ] 
- [ ] Carry forward anything we didn't have time for 

### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

### Prior Meetings
[2022-02-09](https://github.com/hackforla/GreenEarthOS/issues/100#issuecomment-1034071227)

**Comment by ExperimentsInHonesty** [comment]
Item to add to next agenda:
- Review https://github.com/hackforla/GreenEarthOS/issues/106#issuecomment-1049462380

**Comment by ExperimentsInHonesty** [comment]
Next team meeting agenda
- Do workshop https://github.com/hackforla/GreenEarthOS/issues/86

**Comment by rpradheap** [comment]
Agenda - April 20th, 2022

Team

- [ ] Create Brand Value statements in FigJam (https://www.figma.com/file/JImygEUnM6m6giaP9qg3y4/GreenEarthOS-Branding-Exercise?node-id=0%3A1)
    - [ ]  TrustWorthy
    - [ ] Comprehensive
    - [ ] Accessible
    - [ ] Colloborative
    - [ ] Forward thinking

Design Team

- [ ] Work on GreenEarthOS Logo for HackForLAwebsite- https://www.hackforla.org/projects/green-earth-os
- [ ] Image to use in email signature 
        Instructions: Please leave a white space on the top and bottom
                              Please use 1500 x 700 for image size

UX Team

- [ ] Please add contact name and info in column M and N in Open source projects- funder data - https://docs.google.com/spreadsheets/d/1LFResU_pcP5IMwz92dmPQRoKJ4lNa3tvr-_COJiE_hc/edit#gid=0
Instructions
        - Click on the project name and description( Column D in excel) -Click Insights tab on Github project page -  click on contributors (left panel second item ) - select the top contributor-  the contact info will be under the profile picture on the left side.
        - If there is no email , notedown the linkein URL.( You may have to click on personal website to find the details if they can't be found directly via GitHub
- [ ] Please identify top 3 people for Informational Interview
- [ ] Draft a script for email 
- [ ] Draft a script for Linkedin
- [ ] Review draft with team before starting the outreach efforts 
- [ ] Outreach top 3 contacts
- [ ] Review feedback with team about Outreach so we can determine if scripts survey  needs to be changed before continuing outreach to rest


Product Team

- [ ] Please copy the work we today in FigJam to GitHub(https://github.com/hackforla/GreenEarthOS/issues/127) before logging off.

Action Items

**Comment by sanya301** [comment]
## 2022-04-27 Date Agenda and notes

### Roll Call
- [ ] Bonnie
- [ ] Linda
- [ ] Michelle
- [ ] Puneet
- [ ] Raji
- [ ] Sanya
- [ ] Wen

#### Recurring items: 
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each team member
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Sweta](https://github.com/hackforla/GreenEarthOS/issues/assigned/ssweta)
       - [ ] [Joanna](https://github.com/hackforla/GreenEarthOS/issues/assigned/joval604)
       - [ ] [Michelle](https://github.com/hackforla/GreenEarthOS/issues/assigned/miolow)
       - [ ] [Vaisali](https://github.com/hackforla/GreenEarthOS/issues/assigned/vaisali89)

#### New Items
- [ ] 

### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

**Comment by sanya301** [comment]
## 2022-04-27 Date Agenda and notes

### Roll Call
- [ ] Bonnie
- [ ] Linda
- [ ] Michelle
- [ ] Natalia
- [ ] Puneet
- [ ] Raji
- [ ] Sanya
- [ ] Vaisali
- [ ] Wen

#### Recurring items: 
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each team member
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Linda](https://github.com/hackforla/GreenEarthOS/issues/assigned/lkawentel)
       - [ ] [Michelle](https://github.com/hackforla/GreenEarthOS/issues/assigned/miolow)
       - [ ] [Natalia](https://github.com/hackforla/GreenEarthOS/issues/assigned/nat-nn)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Raji](https://github.com/hackforla/GreenEarthOS/issues/assigned/rpradheap)
       - [ ] [Sanya](https://github.com/hackforla/GreenEarthOS/issues/assigned/sanya301)
       - [ ] [Vaisali](https://github.com/hackforla/GreenEarthOS/issues/assigned/vaisali89)
       - [ ] [Wen](https://github.com/hackforla/GreenEarthOS/issues/assigned/wenuxui)

#### Current Items
- [ ] Create Brand Value statements in FigJam (https://www.figma.com/file/JImygEUnM6m6giaP9qg3y4/GreenEarthOS-Branding-Exercise?node-id=0%3A1)
    - [ ]  TrustWorthy
    - [x] Comprehensive
    - [x] Accessible
    - [ ] Colloborative
    - [ ] Forward thinking
- [ ] Update team roster https://docs.google.com/spreadsheets/d/1m-X5vKNoooRV3rP9Ihm6EPQHZ9JiVogNAQBFwII0-oM/edit#gid=0


### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

**Comment by sanya301** [comment]
## 2022-05-11 Date Agenda and notes

### Roll Call
- [ ] Bonnie
- [ ] Linda
- [ ] Michelle
- [ ] Natalia
- [ ] Puneet
- [ ] Raji
- [ ] Sanya
- [ ] Vaisali
- [ ] Wen

#### Recurring items: 
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each team member
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Linda](https://github.com/hackforla/GreenEarthOS/issues/assigned/lkawentel)
       - [ ] [Michelle](https://github.com/hackforla/GreenEarthOS/issues/assigned/miolow)
       - [ ] [Natalia](https://github.com/hackforla/GreenEarthOS/issues/assigned/nat-nn)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Raji](https://github.com/hackforla/GreenEarthOS/issues/assigned/rpradheap)
       - [ ] [Sanya](https://github.com/hackforla/GreenEarthOS/issues/assigned/sanya301)
       - [ ] [Vaisali](https://github.com/hackforla/GreenEarthOS/issues/assigned/vaisali89)
       - [ ] [Wen](https://github.com/hackforla/GreenEarthOS/issues/assigned/wenuxui)

#### Current Items
- [ ] Create Brand Value statements in FigJam (https://www.figma.com/file/JImygEUnM6m6giaP9qg3y4/GreenEarthOS-Branding-Exercise?node-id=0%3A1)
    - [ ]  TrustWorthy
    - [x] Comprehensive
    - [x] Accessible
    - [ ] Colloborative
    - [ ] Forward thinking
- [ ] Update team roster to add Wen https://docs.google.com/spreadsheets/d/1m-X5vKNoooRV3rP9Ihm6EPQHZ9JiVogNAQBFwII0-oM/edit#gid=0


### FYIs

### Notes from meeting

### Task Items

### Items for next weeks Agenda

**Comment by rpradheap** [comment]
- [ ] ## 2022-05-25 Date Agenda and notes

### Roll Call
- [ ] Bonnie
- [x] Linda
- [ ] Michelle
- [x] Natalia
- [x] Puneet
- [x] Raji
- [ ] Sanya
- [x] Vaisali
- [ ] Wen
- [ ] Yao
- [x] Leslie

#### New Member - Introduction
- [ ] Trang Pham


#### Current Items
- [ ] Open 2FA for Wen
- [ ] Create Brand Value statements in FigJam (https://www.figma.com/file/JImygEUnM6m6giaP9qg3y4/GreenEarthOS-Branding-Exercise?node-id=0%3A1)
    - [ ]  TrustWorthy
    - [ ]  Forward thinking
    - [x] Comprehensive
    - [x] Accessible
    - [x] Colloborative
- [ ] Update team roster to add all details for Michelle and Wen https://docs.google.com/spreadsheets/d/1m-X5vKNoooRV3rP9Ihm6EPQHZ9JiVogNAQBFwII0-oM/edit#gid=0

- [ ] Hero Image Review - if Michelle has a new version
- [ ] Review Draft for Informational Interview - Wen
- [ ] Check with Wen on UX meeting time 

#### Recurring items: 
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each team member
       - [ ] [Bonnie](https://github.com/hackforla/GreenEarthOS/issues/assigned/ExperimentsInHonesty)
       - [ ] [Linda](https://github.com/hackforla/GreenEarthOS/issues/assigned/lkawentel)
       - [ ] [Michelle](https://github.com/hackforla/GreenEarthOS/issues/assigned/miolow)
       - [ ] [Natalia](https://github.com/hackforla/GreenEarthOS/issues/assigned/nat-nn)
       - [ ] [Puneet](https://github.com/hackforla/GreenEarthOS/issues/assigned/PLoiya)
       - [ ] [Raji](https://github.com/hackforla/GreenEarthOS/issues/assigned/rpradheap)
       - [ ] [Sanya](https://github.com/hackforla/GreenEarthOS/issues/assigned/sanya301)
       - [ ] [Vaisali](https://github.com/hackforla/GreenEarthOS/issues/assigned/vaisali89)
       - [ ] [Wen](https://github.com/hackforla/GreenEarthOS/issues/assigned/wenuxui)

### FYIs

### Notes from meeting

### Task Items


### Items for next weeks Agenda

Color Palette review with Natalia based on colors matching our brand values
